client/my-sites/customer-home/controller.jsx

```diff
@@ -29,21 +49,21 @@ export function maybeRedirect( context, next ) {
 	}
 
 	const siteId = getSelectedSiteId( state );
-	const options = getSiteOptions( state, siteId );
+	const maybeStalelaunchpadScreenOption = getSiteOptions( state, siteId )?.launchpad_screen;
+
+	if ( maybeStalelaunchpadScreenOption && maybeStalelaunchpadScreenOption !== 'off' ) {
```

Comment:
I wonder if  `=== 'full'` makes more sense than `!== 'off'`? With the possible states of 'off', 'mini', and 'full', 'full' is the only value that should cause a redirect anyways.

Reply:
Makes sense to me 👍 Making the changes now.

Reply:
[f21271b](https://github.com/Automattic/wp-calypso/pull/67967/commits/f21271bb0026d652ce18562f05ffc4d1c01f76e4)